run_unit_tests.sh

```diff
@@ -21,8 +21,9 @@ RUN_TESTS=${RUN_TESTS:-"1"}
 RUN_GPU_TESTS=${RUN_GPU_TESTS:-"0"}
 CHECK_COVERAGE=${CHECK_COVERAGE:-"0"}
 TEST_LOG=${TEST_LOG:-"test.log"}
-debug_bazel_flags=${debug_bazel_flags:-"--config=mp_on_py_on"}
-
+FAIL_LOG=${FAIL_LOG:-"fail.log"}
+if [ -f /etc/redhat-release ] ; then dist="--//:distro=redhat" ; fi
+debug_bazel_flags=${debug_bazel_flags:-"--config=mp_on_py_on $dist"}
```

Comment:
why do we need to manually modify bazel flags during run unit test? why wasnt it required before?

Reply:
@dkalinowski bazel flags were passed from the makefile. It doesn't change. Now there are default values configured to simplify manual execution in the build container